OpenRA.Game/Player.cs

```diff
@@ -258,7 +243,7 @@ public bool IsAlliedWith(Player p)
 			return RelationshipWith(p) == PlayerRelationship.Ally;
 		}
 
-		public Color PlayerRelationshipColor(Actor a)
+		public static Color PlayerRelationshipColor(Actor a)
```

Comment:
I know this is *the* commit message - but *why?* What's the goal here?

Reply:
It should have been static in the first place